test/evergreen.yml

```diff
@@ -1552,7 +1552,7 @@ variables:
       - func: "format test disagg"
         vars:
           # Validate data with a non-layered table.
-          format_args: disagg.page_log=palite disagg.mode=leader ops.verify=1 runs.mirror=1 table1.runs.source=table table1.disagg.enabled=0 runs.tables=2
+          format_args: disagg.page_log=palite disagg.mode=leader ops.verify=1 runs.mirror=1 table1.runs.source=table table1.disagg.enabled=0 runs.tables=2 runs.timer=10:15
```

Comment:
we can remove table here as well. I wish there was away to provide a minimum nuber of tables here, since we wish the first one to be non-disagg.